tests/exists_test.go

```diff
@@ -0,0 +1,33 @@
+package tests
+
+import (
+	"testing"
+	"time"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestExists(t *testing.T) {
```

Comment:
We should add a test case where we are calling `Exists` on an expired key.

Reply:
I see we have a specific test case where we are adding a delay while calling `Exists`. IMO, this should be a different test case.

Reply:
Hi, I will update the test cases in a modular format with each sub-test and details about the scenario.